plugins/proxies/lib_fileservices.ecllib

```diff
@@ -112,4 +112,5 @@ export FileServices := SERVICE : plugin('fileservices'), time
   integer4 GetExpireDays(const varstring lfn) : c,context,entrypoint='fsGetExpireDays';
   SetExpireDays(const varstring lfn, integer4 expireDays) : c,context,entrypoint='fsSetExpireDays';
   ClearExpireDays(const varstring lfn) : c,context,entrypoint='fsClearExpireDays';
+  boolean GetNoCommonDefault() : c,context,entrypoint='fsGetNoCommonDefault';
```

Comment:
I'm not quite sure how this links, probably because it is within an extern "C" so name mangling is disabled.  The function does not have an ICodeContext  * parameter, so it should not have a context attribute in this list.

Reply:
Removed